### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by adam-moss**
Update Travis with the latest version of node, major version releases of node, and iojs.

### CODE REVIEW

**Comment by phated** [comment]
Our only supported platforms will be node 0.10, 0.12, 4, 5 and 6; we will not be supporting specific version of node like 4.2 nor iojs releases.

**Comment by adam-moss** [comment]
Updated as per comment.

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
+  - "node"
```
Comment (phated) [inline .travis.yml]: no need for `node` if all the versions are specified.  We want to be explicit when we support something new (e.g. node 7 when it comes out) but right now this just does another run of node 6.

**Comment by adam-moss** [comment]
Updated :)

**Comment by phated** [comment]
@adam-moss please remove the `node` line.  Since we are being specific about the versions we are supporting on CI, I don't want a replacement for `stable`.  Thanks

**Comment by adam-moss** [comment]
Done.

**Comment by phated** [comment]
Closed by #1692

**Pull request closed without merging**